api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -45,6 +47,18 @@ public void willStart() {
      * @see ModuleActivator#started()
      */
     public void started() {
+        String locationUserPropertyName = Context.getAdministrationService().getGlobalProperty(LocationBasedAccessConstants.REF_APP_LOCATION_USER_PROPERTY_NAME);
+        if(StringUtils.isBlank(locationUserPropertyName)) {
+            Context.getAdministrationService().setGlobalProperty(LocationBasedAccessConstants.REF_APP_LOCATION_USER_PROPERTY_NAME,
```

Comment:
If you expect the value to be LocationBasedAccessConstants.LOCATION_USER_PROPERTY_NAME, then why are you doing a gp for it?

Reply:
The reason was, in our module, we can use this LocationBasedAccessConstants.LOCATION_USER_PROPERTY_NAME. 
But in the RefApp module, it should be decoupled to use another property names for different usages. So I used the RefApp GP to track that value, and for only our module I used this.